app/views/declaration/representative_details.scala.html

```diff
@@ -17,70 +17,91 @@
-                RadioOption("statusCode_indirect", IndirectRepresentative,
-                    messages("supplementary.representative.representationType.indirect"), Some(messages("supplementary.representative.representationType.indirect.hint")))
-            )
+@tariffBody = {
+    @components.tariff_body_partial(
+        messages("supplementary.representative-details.help-item1"),
+        messages("supplementary.representative-details.help-item2")
+    )(tariffBodyDesc)
+  }
+
+@govukLayout(
+    title = Title("supplementary.representative.title"),
+    backButton = Some(BackButton("site.back", Navigator.backLink(RepresentativeDetails, mode)))) {
+
+    @formHelper(action = RepresentativeDetailsController.submitForm(mode), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
+
+        @sectionHeader(messages("supplementary.summary.parties.header"))
+
+        @pageTitle(messages("supplementary.representative.title"), classes = "govuk-heading-xl")
+
+        @govukRadios(Radios(
+            name = "statusCode",
+            fieldset = Some(Fieldset(
+                legend = Some(Legend(
+                    content = Text(messages("supplementary.representative.legend")),
+                    isPageHeading = false,
+                    classes = "govuk-fieldset__legend govuk-fieldset__legend--m"
+                ))
+            )),
+            items = Seq(
+                RadioItem(
+                    id = Some(DirectRepresentative.toString),
+                    value = Some(DirectRepresentative.toString),
+                    content = Text(messages("supplementary.representative.representationType.direct")),
+                    hint = Some(Hint(
+                        content = Text(messages("supplementary.representative.representationType.direct.hint"))
+                    )),
+                    checked = form("statusCode").value.contains(DirectRepresentative.toString)
+                ),
+                RadioItem(
+                    id = Some(IndirectRepresentative.toString),
+                    value = Some(IndirectRepresentative.toString),
+                    content = Text(messages("supplementary.representative.representationType.indirect")),
+                    hint = Some(Hint(
+                        content = Text(messages("supplementary.representative.representationType.indirect.hint"))
+                    )),
+                    checked = form("statusCode").value.contains(IndirectRepresentative.toString)
+                )
+            ),
+            errorMessage = form("value").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*))))
+        ))
+
+        @exportsInputText(
+            field = form("details.eori"),
+            labelKey = "supplementary.representative.eori.legend",
+            hintKey = Some("supplementary.representative.eori.hint"),
+            isPageHeading = true,
+            headingClasses = "govuk-label--m"
         )
 
```

Comment:
There is a GDS toolkit component to do summary details.  See consignment references - 
```        @govukDetails(Details(
            attributes = Map("id" -> "tariffReference"),
            summary = Text(messages("declaration.type.consignmentTariff_summary_text")),
            content = HtmlContent(tariffBody)
        ))
```

Maybe that can be wrapped up in a component.gds component?